From da698db0c20507f0e07492cbe40dbaf1c9053f71 Mon Sep 17 00:00:00 2001
From: Thomas Devoogdt <thomas@devoogdt.com>
Date: Sun, 12 Nov 2023 09:58:05 +0100
Subject: [PATCH] cairo-ft-private.h: fix missing FT_Color error

In file included from ../src/cairo-colr-glyph-render.c:37:
../src/cairo-ft-private.h:87:30: error: unknown type name 'FT_Color'
   87 |                              FT_Color               *palette,
      |                              ^~~~~~~~

Upstream: https://gitlab.freedesktop.org/cairo/cairo/-/issues/792
Signed-off-by: Thomas Devoogdt <thomas@devoogdt.com>
---
 src/cairo-ft-private.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/cairo-ft-private.h b/src/cairo-ft-private.h
index 836f7e523..6b0e30223 100644
--- a/src/cairo-ft-private.h
+++ b/src/cairo-ft-private.h
@@ -43,6 +43,8 @@
 
 #if CAIRO_HAS_FT_FONT
 
+#include FT_COLOR_H
+
 CAIRO_BEGIN_DECLS
 
 typedef struct _cairo_ft_unscaled_font cairo_ft_unscaled_font_t;
-- 
2.34.1

